cosmo_tester/test_suites/summary/summary_test.py

```diff
@@ -404,26 +365,28 @@ def test_subfield_summary_deployments(prepared_manager):
         {
             u'by blueprint_id': [
                 {u'blueprint_id': u'multivm', u'deployments': 3},
-                {u'blueprint_id': u'small', u'deployments': 1},
-                {u'blueprint_id': u'relations', u'deployments': 2}
+                {u'blueprint_id': u'relations', u'deployments': 2},
```

Comment:
Any reason for these being reordered here?

Reply:
This way the list is ordered by `blueprint_id`, as the `results` list.

Reply:
Fair enough.